Explorer/Assets/Scripts/Global/Dynamic/RealmNavigator.cs

```diff
@@ -27,6 +38,9 @@ public class RealmNavigator : IRealmNavigator
         private readonly IRoomHub roomHub;
         private readonly IRemoteEntities remoteEntities;
         private readonly ObjectProxy<World> globalWorldProxy;
+        private readonly LandscapePlugin landscapePlugin;
```

Comment:
I think it's a bad idea to reference a plugin that belongs to the higher level of abstraction, it creates a smelling precedence: use only the dependencies you need